java/runtime/src/main/resources/ray.default.conf

```diff
@@ -38,6 +38,14 @@ ray {
            // key1: "value11"
            // key2: "value22"
          }
+
+         // The packages that for this job. Dashboard agent will download from the
+         // urls and then the workers of this job will add them to classpath.
+         // Note that it supports both jar packages and zip packages.
+         "jars": [
+            // "https://my_host.a.jar",
+            // "https://my_host.b.jar"
```

Comment:
```suggestion
            // "https://my_host/b.jar"
```

Reply:
Due to all CI cases get green, I'm merging this PR and then change the typo later.